admin_guide/encrypting_data.adoc

```diff
@@ -276,33 +276,19 @@ on all servers.
 If using a single API server, you can skip this step.
 ====
 +
-ifdef::openshift-origin[]
-----
-# systemctl restart origin-master-api origin-master-controllers
-----
-endif::[]
-ifdef::openshift-enterprise[]
 ----
-# systemctl restart atomic-openshift-master-api
+# systemctl master-restart api
 ----
-endif::[]
 
 . Make the new key the first entry in the `keys` array so that it is used for
 encryption in the configuration.
 
 . Restart all API servers to ensure each server now encrypts using the new
 key.
 +
-ifdef::openshift-origin[]
-----
-# systemctl restart origin-master-api origin-master-controllers
-----
-endif::[]
-ifdef::openshift-enterprise[]
 ----
-# systemctl restart atomic-openshift-master-api
+# systemctl master-restart api
```

Comment:
@mburke5678 Again want to make sure that both origin and enterprise now share the command?